src/taskpane/server.js

```diff
@@ -1789,6 +1985,24 @@ function getHierarchicalParentId(indent, previousEntries) {
   return -1;
 }
 
+// returns the correct parentId for the relevant group position by looping back through the list of entries
+// returns -1 if no match found

```

Comment:
why does it return -1 as opposed to false? What is special about -1?

Reply:
There's nothing special actually - I just wanted to keep the original 'log' value, as it was originally created as -1. I can change that to 'false', it won't create any problem.